benchmarks/operator_benchmark/pt/qbatchnorm_test.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import operator_benchmark as op_bench
+import torch
+
+
+"""Microbenchmarks for quantized batchnorm operator."""
+
+batchnorm_configs_short = op_bench.config_list(
+    attr_names=["M", "N", "K"],
+    attrs=[
+        [1, 256, 3136],
+    ],
+    cross_product_configs={
+        'device': ['cpu'],
+        'dtype': (torch.qint8,),
+    },
+    tags=["short"]
+)
+
+
+class QBatchNormBenchmark(op_bench.TorchBenchmarkBase):
+    def init(self, M, N, K, device, dtype):
+        # Note: quantized implementation requires rank 4, which is why we
+        # add a 1 as the last dimension
+        self.input_one = torch.rand(M, N, K, 1, device=device, requires_grad=self.auto_set())
```

Comment:
Can't we instead pass in 4 dimensions (NCHW) to this function?

Reply:
currently these match the floating point BN benchmark exactly for a fair comparison.  Open to NCHW but would be good to have the fp and q match - we can do this in a separate PR!